docs/style-guide.adoc

```diff
@@ -1,6 +1,14 @@
+:numbered:
+:toc:
+:toc-placement: macro
+
 = The Cypher Style Guide
 
-Style guide for writing Cypher code.
+toc::[]
+
+This is the style guide for the Cypher language, in the context of its standardization through the openCypher project.
+In this document you will find rules for how to use the language, intended to help construct a consistent and portable usage of the language across many use cases and implementations.
```

Comment:
"In this document you will find rules for how to use the language" -> "This document contains rules for how to use the language"